client/src/client.rs

```diff
@@ -873,6 +875,29 @@ impl Client {
         self.build_query(request).execute()
     }
 
+    /// Query API entry point using cursor.
+    ///
+    /// # Errors
+    /// Fails if sending request fails
+    pub fn request_with_cursor<O>(&self, cursor: ForwardCursor) -> QueryResult<O::Target>
```

Comment:
shouldn't you put this into `client/src/query_builder.rs`?

Reply:
please also remove `request_with_filter_and_pagination_and_sorting`. It must have been left in accidentally

Reply:
And it uses some private parts of `Client` so looks like we have to keep it

Reply:
> shouldn't you put this into client/src/query_builder.rs?

I just tried and I don't think so. There is nothing to customize in a query by cursor, so nothing to build. And for simple queries we have `client.request()`. So I think having `client.request_with_filter()` is also ok

Reply:
> I'll make it `pub(crate)`

this is proper solution. This should not be exposed